src/lib/brokerHandler.ts

```diff
@@ -45,3 +45,9 @@ export const handleGetTrades = async () => {
   });
   return response;
 };
+
+export const getWalletBalance = async () => {
+  const resp = await fetch("/api/broker/account/balance");
+  const data = await convertResult(resp);
+  return data.meta.balance;
```

Comment:
Perhaps add some validation on the "status: "OK"/"NOK" " here, or sanity check that we have the balance prop on meta?